build-tools/scripts/DotNet.targets

```diff
@@ -26,6 +26,12 @@
         Projects="$(_Root)build-tools\create-dotnet-msi\create-dotnet-msi.csproj"
         Properties="Configuration=$(Configuration)"
     />
+    <MSBuild
+        Condition=" $([MSBuild]::IsOSPlatform('osx')) "
+        Targets="Restore;Build"
+        Projects="$(_Root)build-tools\create-dotnet-pkg\sign-dotnet-pkg.proj"
+        Properties="Configuration=$(Configuration)"
+    />
```

Comment:
Does this need to check some MSBuild property to skip it? Would it attempt to sign packages on Mac locally otherwise?

Reply:
I'll add some info to the description about this. None of the MicroBuild stuff will run unless the `MicroBuildOverridePluginDirectory` or `MicroBuildPluginDirectory` properties are set and pointing to a valid directory. Building this project locally will only codesign and harden the mach-o files that are going in the .pkg file, which can be done without specifying a signing identity (no Apple certs required on disk).